subsys/bluetooth/controller/ll_sw/ull_internal.h

```diff
@@ -9,6 +9,11 @@ static inline u8_t ull_ref_inc(struct ull_hdr *hdr)
 	return ++hdr->ref;
 }
 
+static inline u8_t ull_ref_decr(struct ull_hdr *hdr)
```

Comment:
@mped-oticon Minor nitpicking: shouldn't it be called ull_ref_dec if we are going for consistency with ull_ref_inc?

Reply:
done